CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/purecloudlabs/genesys-cloud-streaming-client/compare/v13.2.9...HEAD)
+### Added
+* [PCM-1715](https://inindca.atlassian.net/browse/PCM-1715) - Added ability to end sessions from the client.
```

Comment:
This may be a NIT, but could this note be a little more clear? Technically, we already have the ability to end a session from the client. This PR is more overriding the current stanza implementation of ending sessions, and lets the backend end the PC or it ends it after 2 seconds.